packages/centreon-ui/src/MenuSkeleton/index.tsx

```diff
@@ -1,31 +1,50 @@
-import { useTheme, alpha } from '@mui/material';
+import clsx from 'clsx';
+import { equals } from 'ramda';
+
+import { SkeletonProps, useTheme, alpha } from '@mui/material';
 import makeStyles from '@mui/styles/makeStyles';
 
-import LoadingSkeleton from '../LoadingSkeleton';
+import { LoadingSkeleton } from '@centreon/ui';
 
 const useStyles = makeStyles((theme) => ({
   skeleton: {
     backgroundColor: alpha(theme.palette.grey[50], 0.4),
-    margin: theme.spacing(0.5, 2, 1, 2),
+    borderRadius: 5,
+    margin: theme.spacing(1, 1, 1, 2),
   },
 }));
 
 interface Props {
   animate?: boolean;
-  width?: number;
+  className?: string;
+  height?: number | string;
+  variant?: SkeletonProps['variant'];
+  width?: number | string;
 }
 
-const MenuLoader = ({ width = 15, animate = true }: Props): JSX.Element => {
+const MenuLoader = ({
+  width = 15,
+  height = 5,
+  className,
+  variant = 'rectangular',
+  animate = true,
+}: Props): JSX.Element => {
   const theme = useTheme();
   const classes = useStyles();
 
   return (
     <LoadingSkeleton
       animation={animate ? 'wave' : false}
-      className={classes.skeleton}
-      height={theme.spacing(5)}
-      variant="text"
-      width={theme.spacing(width)}
+      className={clsx(classes.skeleton, className)}
+      height={
+        equals(typeof height, 'string')
+          ? height
+          : theme.spacing(height as number)
+      }
+      variant={variant}
+      width={
+        equals(typeof width, 'string') ? width : theme.spacing(width as number)
+      }
```

Comment:
We want to define width and height using the default Material UI spacing
```suggestion
      height={
        theme.spacing(height as number)
      }
      variant={variant}
      width={
        theme.spacing(width as number)
      }
```